refinery/data_set_manager/search_indexes.py

```diff
@@ -166,6 +122,10 @@ def prepare(self, node):
             file_store_item = None
             download_url = NOT_AVAILABLE
         else:
+            data['filetype_Characteristics' + NodeIndex.GENERIC_SUFFIX] = \
```

Comment:
once fileStoreItem confirmed, we set filetype. It's the same filetype as in the file browser